src/hardware/vga_draw.cpp

```diff
@@ -1150,9 +1150,9 @@ void VGA_SetupDrawing(Bitu /*val*/) {
 			}
 		}
 
-		/* Check for 8 for 9 character clock mode */
+		/* Check for 8 for 9 character clock modes */
```

Comment:
In reference to the comment - generally I agree, but I suspect (not sure) word "master" was supposed to have the same meaning as for previous commits in OPL and gameblaster: ISA reference clock.

Reply:
Thanks @dreamer ; after looking into this, the closest description I can find is that these are sync frequencies.  I don't think they are related to ISA bus frequency.

For example, look a bit higher up and we see `clock` is assigned from `get_clock()`:

``` c++
		if (svga.get_clock) {
			clock = svga.get_clock();
```

`get_clock()` is a function pointer to Paradise, ET4000, specific clock functions:

``` text
src/hardware/vga_paradise.cpp:  svga.get_clock = &GetClock_PVGA1A;
src/hardware/vga_s3.cpp:        svga.get_clock = &SVGA_S3_GetClock;
src/hardware/vga_tseng.cpp:static Bitu get_clock_index_et4k() {
src/hardware/vga_tseng.cpp:     return et4k.clockFreq[get_clock_index_et4k()];
src/hardware/vga_tseng.cpp:     svga.get_clock = &GetClock_ET4K;
src/hardware/vga_tseng.cpp:static Bitu get_clock_index_et3k() {
src/hardware/vga_tseng.cpp:     return et3k.clockFreq[get_clock_index_et3k()];
src/hardware/vga_tseng.cpp:     svga.get_clock = &GetClock_ET3K;
```

Thse functions return small values (and don't seem to be connected to ISA clock speeds). For example, here's one of them: 

```
src/hardware/vga_paradise.cpp:Bitu GetClock_PVGA1A() {
src/hardware/vga_paradise.cpp-  return pvga1a.clockFreq[(vga.misc_output >> 2) & 3];
```

Where `vga.misc_output` holds small values:

``` text
src/ints/int10_modes.cpp:       Bit8u misc_output=0x2 | (mono_mode ? 0x0 : 0x1);
src/ints/int10_modes.cpp:               if ((CurMode->vdispend==350) && (mode!=0xf)) misc_output|=0x4;
src/ints/int10_modes.cpp:               if ((CurMode->type==M_TEXT) && (CurMode->cwidth==9)) misc_output|=0x4;
src/ints/int10_modes.cpp:               misc_output|=0x60;
src/ints/int10_modes.cpp:               misc_output|=0xe0;
src/ints/int10_modes.cpp:               misc_output|=0xa0;
src/ints/int10_modes.cpp:               misc_output|=0x60;
src/ints/int10_modes.cpp:       IO_Write(0x3c2,misc_output);            //Setup for 3b4 or 3d4
src/ints/int10_modes.cpp:                               misc_output|=0xc0;      //480-line sync
src/ints/int10_modes.cpp:                       misc_output|=0x0c;              //Select clock 3 
src/ints/int10_modes.cpp:       IO_Write(0x3c2,misc_output);
```

Reply:
Will try a bit more to word this better :+1:

Reply:
Thanks @dreamer.  Reworded based on details here: https://wiki.osdev.org/VGA_Hardware

![2021-07-08_18-06](https://user-images.githubusercontent.com/1557255/125008359-3865d700-e017-11eb-8163-f081a4061681.png)